docs/specification/1.0.md

```diff
@@ -441,46 +441,61 @@ The EHR’s authorization server is responsible for enforcing restrictions on th
+
+The transaction the EHR uses to request authorization MUST include the following parameters:
+ 
+ Field | Description
+ ----- | --------
+ response_type | Fixed value:  code
+ client_id | EHR requesting authorization
+ subject | client_id of authorized accessor (CDS Service) for which the access token is being requested 
+ scope | REQUIRED | The access being requested 
+ aud | REQUIRED | The FHIR server (fhirServer) from which resources will be retrieved
+ 
+The `scope` value SHOULD contain the minimum necessary scopes the CDS Service requires to compute its logic.  As the CDS Service executes on behalf of a user, the data to which the CDS Service is given access SHOULD BE limited to the same restrictions and authorizations afforded the current user. As such, the access token SHALL BE scoped to:
+
+- The CDS Service being invoked
+- The current user
+
+If the authorization request is granted, the authorization server returns an authorization code that the EHR exchanges for a bearer access token.  The access token is a string of characters as defined in [RFC6749](https://tools.ietf.org/html/rfc6749) and [RFC6750](https://tools.ietf.org/html/rfc6750) and should have high entropy.  The token is essentially a private message that the authorization server passes to the FHIR Server, telling the FHIR server that the “message bearer” has been authorized to access the specified resources.  Defining the format and content of the access token is left up to the organization that issues the access token and holds the requested resource.
 
-With CDS Hooks, the EHR provides the access token directly in the request to the CDS Service. Thus, the CDS Service does not need to request the token from the authorization server as a SMART app would. This is done purely for performance reasons as the authorization code grant model in OAuth 2 involves several HTTPS calls and redirects. In contrast to a SMART app, a CDS Service may be invoked many times during a workflow. Going through the authorization code grant model on every hook invocation would likely result in a poorly performing CDS Service due to the authorization overhead.
+The authorization server’s response MUST indicate that the access token MAY NOT be cached or otherwise persisted in data storage.  The expiry time for which the token remains value SHOULD BE set as short as possible to provide the requested decision support.
 
 ```json
 {
   "fhirAuthorization" : {
     "access_token" : "some-opaque-fhir-access-token",
-    "token_type" : "Bearer",
+    "token_type" : "bearer",
     "expires_in" : 300,
-    "scope" : "patient/Patient.read patient/Observation.read"
+    "scope" : "patient/Patient.read patient/Observation.read",
+    "subject" : "cds-service4",
+    "aud" : "http://fhirServer.ehr.com
```

Comment:
We don't use the `aud` parameter. We discussed adding in the JWT but ultimately decided against it since every CDS Service request has a `fhirServer` field which is duplicative here.